### ISSUE TITLE
l4d2 sound manupilation

### ISSUE THREAD

**Issue by markgs** [Attano/L4D2-Competitive-Framework#12]
hello, the version of this plugin sp file is 1.0 while the promod 4.5.2 release is version 2.0. is/are there any differences?

(https://github.com/Attano/L4D2-Competitive-Framework/blob/master/addons/sourcemod/scripting/l4d2_sound_manipulation.sp)

**Comment by Attano** [comment]
Yes, I haven't uploaded the promod version because it's broken. There were numerous reports about survivor vocalizations not being played, some of which can be critical.
